spec/factories/cacher.rb

```diff
@@ -0,0 +1,12 @@
+require './lib/renuo/bin-check/command_result'
+FactoryGirl.define do
```

Comment:
you know that you don't need to use factory girl for _everything_? i mean it looks nice this way, but if creating a simple ruby object is simpler, than you can just use the constructor of a class ;)